internal/acctest/config/pingoneconnection/ping_one_connection_test.go

```diff
@@ -0,0 +1,161 @@
+package acctest_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-framework/providerserver"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/acctest"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/provider"
+)
+
+var pingOneConnectionId = "myPingOneConnectionId"
+var pingOneConnectionName = "myPingOneConnectionName"
+var credentialData = os.Getenv("PF_TF_ACC_TEST_PING_ONE_CONNECTION_CREDENTIAL_DATA")
+
+// Attributes to test with. Add optional properties to test here if desired.
+type pingOneConnectionResourceModel struct {
+	name        string
+	description string
+	active      bool
+	credential  string
+}
+
+func TestAccPingOneConnection(t *testing.T) {
+	resourceName := "myPingOneConnection"
+	initialResourceModel := pingOneConnectionResourceModel{
+		name:       pingOneConnectionName,
+		credential: credentialData,
+	}
+	updatedResourceModel := pingOneConnectionResourceModel{
+		name:        pingOneConnectionName,
+		credential:  credentialData,
+		description: "Updated description",
+		active:      false,
+	}
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			acctest.ConfigurationPreCheck(t)
+			if credentialData == "" {
+				t.Fatal("PF_TF_ACC_TEST_PING_ONE_CONNECTION_CREDENTIAL_DATA must be set for acceptance tests")
+			}
+		},
+		ProtoV6ProviderFactories: map[string]func() (tfprotov6.ProviderServer, error){
+			"pingfederate": providerserver.NewProtocol6WithError(provider.NewTestProvider()),
+		},
+		CheckDestroy: testAccCheckPingOneConnectionDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccPingOneConnection(resourceName, initialResourceModel),
+				Check:  testAccCheckExpectedPingOneConnectionAttributes(initialResourceModel),
```

Comment:
Based on Patrick's recommendations, we should add some checks for computed attributes here

Reply:
added